### PR TITLE
Add IoT to the CSV

### PR DESCRIPTION
**Pull request by ctron**

### CODE REVIEW

**Comment by ctron** [comment]
@enmasse-ci run tests profile=shared-iot

**Comment by ctron** [comment]
@enmasse-ci run tests profile=shared-iot

**Comment by ctron** [comment]
@enmasse-ci run tests profile=shared-iot

**Comment by ctron** [comment]
@enmasse-ci run tests profile=shared-iot

**Comment by ctron** [comment]
@enmasse-ci run tests profile=shared-iot

**Comment by ctron** [comment]
@enmasse-ci run tests profile=shared-iot

**Comment by ctron** [comment]
@enmasse-ci run tests profile=shared-iot

**Comment by ctron** [comment]
@enmasse-ci run tests profile=shared-iot

**Comment by ctron** [comment]
@enmasse-ci run tests profile=shared-iot

**Comment by ctron** [comment]
@enmasse-ci run tests profile=shared-iot

**Comment by ctron** [comment]
@enmasse-ci run tests profile=shared-iot

**Comment by ctron** [comment]
@enmasse-ci run tests profile=shared-iot

File: templates/csv/enmasse.clusterserviceversion.yaml [hunk 1]
```diff
@@ -176,6 +176,34 @@ metadata:
...
+                },
+                "addresses": {
+                  "command": {
+                    "plan": "standard-small-anycast"
+                  },
+                  "event": {
+                    "plan": "standard-small-queue"
+                  },
+                  "telemetry": {
+                    "plan": "standard-small-anycast"
```
Comment (lulf) [inline templates/csv/enmasse.clusterserviceversion.yaml]: One thing to think about here perhaps is that these plans will not necessarily exist, unless the user is creating them by this name. With OLM there are no default plans installed so the documentation should be clear that the plans must be installed (maybe it already is).

Comment (ctron) [reply templates/csv/enmasse.clusterserviceversion.yaml]: Done.

**Comment by ctron** [comment]
I think we are fine with ansible, as we only deploy some YAML files with ansible. And that hasn't changed. The YAML will pre-create the service account, and ansible will call those. Only in the OLM case, the service account is part of the CSV now.

I think we need some work in the docs in any case. I added a section in the CSV, adding some highlights, and pointing to the documentation.

**Comment by ctron** [comment]
@enmasse-ci run tests profile=shared-iot

**Comment by ctron** [comment]
@enmasse-ci run tests profile=shared-iot

**Pull request merged**